docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
       context: .
       target: dev
     env_file:
-      - path: .env
+      - path: .docker-compose-env
```

Comment:
Question: Why does this need to be a different `.env` file?

Reply:
The initial idea here was to allow to distinguish between the local and containerised config files and allow them to run with different settings. On second thought it's probably mostly pointless as you could just overwrite the setting in the `environment` block.